### ISSUE TITLE
Instance has failed at least the UnhealthyThreshold number of health checks consecutively.

### ISSUE THREAD

**Issue by angelllllll** [ryanmurakami/pizza-luvrs#8]
Hi ,

I get the above error for my instances when i have created a cloud stack.

I have checked the security group  rules and the instances are running but they show as out of service in the newly created load balancer page.

Please help

**Comment by angelllllll** [comment]
Similar issue when using elastic beanstalk also.

2017-05-15 21:44:38 UTC-0700	ERROR	The EC2 instances failed to communicate with AWS Elastic Beanstalk, either because of configuration problems with the VPC or a failed EC2 instance. Check your VPC configuration and try launching the environment again.
2017-05-15 21:44:38 UTC-0700	ERROR	Stack named 'awseb-e-pi53mmp2yg-stack' aborted operation. Current state: 'CREATE_FAILED' Reason: The following resource(s) failed to create: [AWSEBInstanceLaunchWaitCondition].

**Comment by jasalazar** [comment]
Did anybody come up with a solution for this?

I could successfully deploy an EBS app the first time, but now I always get this same error **[AWSEBInstanceLaunchWaitCondition].**

**Comment by Yamo93** [comment]
I get the same problem. Is there any solution for this?

**Comment by davew9999** [comment]
I also got the same problem. The box in the target group was saying unhealthy. I hacked around and managed to "fix" it by connecting to the box via ssh and running `npm start`. I don't know why this was needed as according to Elastic Beanstalk it will run `npm start` for you if you leave 'Node command' blank.

> Command to start the Node.js application. If an empty string is specified, app.js is used, then server.js, then "npm start" in that order.

The box then reported as healthy on the next health check and the site loaded. The whole site wasn't working however, I could load the homepage but not load a pizza or login. 

I gave up at this point!
